changelog/4358.breaking.rst

```diff
@@ -0,0 +1 @@
+`:func:~sunpy.net.helio.HECClient.time_query` is removed; ``search`` method defined on the client is expected to be used instead.
```

Comment:
Can you check if ":func:" should in the backticks and if it should be ":func:" or ":meth":?